x/evm/legacy/params.go

```diff
@@ -0,0 +1,163 @@
+package legacy
```

Comment:
this is basically a copy of params.go before adding the new field, right?

Reply:
correct!